mods/seasonal/statuses.js

```diff
@@ -84,13 +84,28 @@ exports.BattleStatuses = {
 			this.add('c|@grimAuxiliatrix|∠( ᐛ 」∠)＿');
 		},
 	},
+	himynamesl: {
+		exists: true,
```

Comment:
needs `nocopy: true,` so it dosent get copied by baton pass, ive set all the other ones and will push once i fix the set i'm working on,